repos/system_upgrade/el7toel8/actors/vimcheck/actor.py

```diff
@@ -0,0 +1,82 @@
+from leapp.actors import Actor
+from leapp.libraries.common.reporting import report_generic
+from leapp.models import InstalledRedHatSignedRPM, VimModel, Report
+from leapp.tags import ChecksPhaseTag, IPUWorkflowTag
+
+vimconfigs = {
+              'vim-minimal': '/etc/virc',
+              'vim-enhanced' : '/etc/vimrc'
+              }
+
+changes = [
+            'let skip_defaults_vim=1',
+            'set t_BE='
+          ]
```

Comment:
changes is not used anywhere in the actor. Additionally you define this object 2x which doesn't look good to me. Please, remove unsued code. Additionally, it's not needed those variables out of the class.